fang_examples/simple_async_worker/src/lib.rs

```diff
@@ -69,3 +70,84 @@ impl AsyncRunnable for MyFailingTask {
         }
     }
 }
+
+#[derive(Serialize, Deserialize)]
+#[serde(crate = "fang::serde")]
+pub struct MyCronTask {
```

Comment:
maybe let's create a new example project for the cron task?

Reply:
[Comments adressed](https://github.com/ayrat555/fang/pull/71/commits/e32b0fe5b77adb1ff9ec8a9453335b37a00bfccc)